test/bookstore/gke/nginx.conf

```diff
@@ -42,7 +42,7 @@ http {
       # Begin Endpoints v2 Support
       endpoints {
         on;
-        server_config /etc/nginx/endpoints/server_config.pb.txt;
+        server_config /etc/nginx/server_config.pb.txt;
```

Comment:
could you also add

api /etc/nginx/endpoints/service.json

back.  I bet customer custom nginx config have that one too.   We want to test it to make sure it works.